### ISSUE TITLE
Add Barb, Meditate Focus, to known lockpick_buffs. 

### ISSUE THREAD

**Issue by jayRyan24** [rpherbig/dr-scripts#777]
In pick.lic

```
  def do_buffs(buffs)
    buffs['spells'].each do |spell|
      echo "Buffing: #{spell}" if UserVars.lockpick_debug
      buff(spell, @settings)
    end

    buffs['khri']
      .map { |name| "Khri #{name}" }
      .each { |name| activate_khri?(@settings.kneel_khri, name) }

    buffs['meditate'].each do |name|
      fput("meditate #{name}")
    end
  end
```

**Comment by rpherbig** [comment]
Do you think it's worth adding a `sit_meditate` setting (for those with Yogi)?

Do all meditates end with `You feel a jolt as your vision snaps shut.`? I think we'd have to wait for some trigger that the meditation has taken effect, or risk starting the pick process early (which I think would interrupt the meditation).

**Comment by jayRyan24** [comment]
Came here to talk about this. Tried to run it and wasn't sitting, so the meditation failed. I added a simple fput("sit") before the meditate, I guess we could handle it by adding something like you mentioned and/or a method like the khri kneel. 

Yup, all the meditations end with "You feel a jolt as your vision snaps shut."

**Comment by jayRyan24** [comment]
Resolved with #1223
